src/machine.rs

```diff
@@ -178,6 +178,20 @@ impl OlmMachine {
         future_to_promise(async move { Ok(me.display_name().await?) })
     }
 
+    /// Mark all tracked users as dirty.
+    ///
+    /// All users *whose device lists we are tracking* are flagged as needing a
+    /// key query. Users whose devices we are not tracking are ignored.
+    #[wasm_bindgen(js_name = "markAllTrackedUsersAsDirty")]
+    pub fn mark_all_tracked_users_as_dirty(&self) -> Promise {
```

Comment:
by way of making sure that this gives an ergonomic interface on the JS side, could you add something to `tests/machine.test.ts` that calls this? Doesn't have to do anything fancy, I'd just like us to call the method.